modules/lotamePanoramaIdSystem.js

```diff
@@ -189,7 +189,15 @@ export const lotamePanoramaIdSubmodule = {
       if (consentData && utils.isBoolean(consentData.gdprApplies)) {
         queryParams.gdpr_applies = consentData.gdprApplies;
         if (consentData.gdprApplies) {
```

Comment:
gdprApplies also gets populated by the cmp. Thus, it won't be present for this case where we look in the cookies. So we should remove this if statement